devito/compiler.py

```diff
@@ -244,8 +244,16 @@ def jit_compile(ccode, basename, compiler=GNUCompiler):
     :param compiler: The toolchain used for compilation. GNUCompiler by default.
     :return: Path to compiled lib
     """
+
     src_file = "%s.%s" % (basename, compiler.src_ext)
-    lib_file = "%s.%s" % (basename, compiler.lib_ext)
+    from sys import platform
```

Comment:
Unless there is a good reason to hide it here, the import should to go the top of the file.